build/repo.targets

```diff
@@ -1,9 +1,11 @@
 <Project>
   <ItemGroup>
     <!-- Optimal ThreadCount for current hardware -->
-    <Scenarios Include="-n Plaintext --kestrelThreadCount 2" />
-    
+    <Scenarios Include="-n Plaintext --kestrelTransport Libuv --kestrelThreadCount 2" />
+    <Scenarios Include="-n Plaintext --kestrelTransport RHLinux --kestrelThreadCount 2" />
```

Comment:
I added one scenario with kestrelThreadCount and one without.
The one with is for the threadpool dispatch, the one without is for the inline dispatch.